### PR TITLE
Pretty print dataclass

### PR DESCRIPTION
**Pull request by danielui**
Created super class for all dataclasses, to change data printing style. Example below:
```
Class name: Device
  deviceId: mock_ip
  host-name: mock_host
  local-system-ip: mock_ip
  connectedVManages: ['192.168.0.1']
  device-model: vedge-cloud
  board-serial: None
  vedgeCertificateState: None
  chasisNumber: None
  uuid: mock_uuid
  status: normal
  personality: Personality.EDGE
  reachability: Reachability.REACHABLE
  memUsage: 1.0
```

### CODE REVIEW

File: vmngclient/dataclasses.py [hunk 1]
```diff
@@ -3,14 +3,23 @@
...
 from vmngclient.utils.personality import Personality
 from vmngclient.utils.reachability import Reachability
 
 
+class DataclassSettings:
+    def __str__(self):
+        return (
+            f'Class name: {self.__class__.__name__}'
+            + "\n"
+            + "\n".join(f'  {attribute[0]}: {attribute[1]}' for attribute in asdict(self).items())
```
Comment (tomaszwarda) [inline vmngclient/dataclasses.py:17]: maybe make it look like in issue suggestion?
```python
Device(
    personality='vbond', 
    id='1.1.1.1'
)
```

```suggestion
            f"{self.__class__.__name__}(\n"
            + "\n".join(f"  {attribute[0]}: {attribute[1]}" for attribute in asdict(self).items())
            + "\n)"
```

Comment (tehAgitto) [reply vmngclient/dataclasses.py:17]: Have you tried
```python
json.dumps(self, indent=4)
```

Comment (danielui) [reply vmngclient/dataclasses.py:17]: Couldn't dump enum to json

**Review by sbasan — DISMISSED** [review]
lgtm

**Review by tomaszwarda — DISMISSED** [review]

File: vmngclient/dataclasses.py [hunk 2]
```diff
@@ -3,14 +3,23 @@
...
 from vmngclient.utils.reachability import Reachability
 
 
+class DataclassBase:
+    def __str__(self):
+        return (
+            f"{self.__class__.__name__}(\n"
+            + "\n".join(f"    {attribute[0]}: {attribute[1]}," for attribute in asdict(self).items())
+            + "\n)"
+        )
```
Comment (tehAgitto) [inline vmngclient/dataclasses.py:18]: Did you try 
```python
json.dumps(asdict(self), indent=4)
```

**Review by tomaszwarda — APPROVED** [review]

**Review by sbasan — APPROVED** [review]
lgtm

**Pull request merged**